app/assets/javascripts/application.js

```diff
@@ -229,28 +229,32 @@ function filter_by_level(item){
 
   // Note that class names don't map to log level names (label-info == notice)
   if(level == 'info'){
-    $('.label-info').closest('tr').show();
-    $('.label-default').closest('tr').show();
-    $('.label-warning').closest('tr').show();
-    $('.label-danger').closest('tr').show();
+    $('.label-info.result-filter-tag').closest('tr').show();
+    $('.label-default.result-filter-tag').closest('tr').show();
```

Comment:
There may be multiple columns in a row that have `label-{tag}`. I want to show only rows that have both `label-{tag}` and `result-filter-tag` classes. I choose on which column with labels to filter by adding `result-filter-tag`.